source/catalog.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body>
+  <header class="page-header">
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="img/icons/logo-mobile.svg" width="33" height="38"
+        alt="Логотип Catenergy.">
+      <img class="page-header__logo-image" src="img/icons/cat_energy.svg" width="101" height="18" alt="Cat Energy.">
+    </a>
```

Comment:
Я бы все же рекомендовал создать отдельный блок для лого и примиксовать его:
```suggestion
    <a class="page-header__logo logo">
      <img class="logo__icon" src="img/icons/logo-mobile.svg" width="33" height="38"
        alt="" aria-hidden="true">
      <img class="logo__image" src="img/icons/cat_energy.svg" width="101" height="18" alt="Cat Energy.">
    </a>
```